### PR TITLE
Rebuild fontconfig 2.14.1 with libxml2 2.10

### PR DESCRIPTION
**Pull request by skupr-anaconda**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -38,10 +38,9 @@ requirements:
     - python  # [win]
     - meson   # [win]
     - ninja   # [win]
-
   host:
     - freetype
-    - libxml2
+    - libxml2 {{ libxml2 }}
```
Comment (katietz) [inline recipe/meta.yaml]: please replace jinja2 libxml2 by the version directly, and remove the entry from recipe's cbc.
additionally make sure an abs.yaml file is turning off overdepending check

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: I replaced  jinja2 libxml2 by the version directly. abs.yaml already exists with disabling overdepending checks

Comment (cbouss) [reply recipe/meta.yaml]: FYI overdepending is not enabled on prefect anymore
https://github.com/anaconda-distribution/rocket-platform/blob/main/cookbook/flows/config.yaml#L18

File: abs.yaml [hunk 2]
```diff
@@ -1,4 +1,3 @@
 # the conda-build parameters to use for disabling --skip-existing
 build_parameters:
   - "--suppress-variables"
-
```
Comment (M-Waszkiewicz-Anaconda) [inline abs.yaml:4]: just leave the file unchanged

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Thanks. I forgot about that. Added to my bookmarks.

**Review by M-Waszkiewicz-Anaconda — APPROVED** [review]
LGTM

**Review by katietz — APPROVED** [review]

**Pull request merged**